src/foam/dao/Relationship.js

```diff
@@ -428,7 +436,9 @@ return junction`
       returns: 'foam.dao.DAO',
       javaCode: 'return getDao();',
       swiftCode: 'return dao!',
-      code: function getDAO() { return this.dao; }
+      code: function getDAO() {
```

Comment:
The original seems better.

Reply:
All the changes I made to the formatting was to satisfy the linter which is based on the rules defined by us. I can change it back.